x/ecocredit/client/tx.go

```diff
@@ -297,8 +298,51 @@ Example JSON:
 // TxSendCmd returns a transaction command that sends credits from one account
 // to another.
 func TxSendCmd() *cobra.Command {
+	var retireTo string = ""
 	cmd := &cobra.Command{
-		Use:   "send [recipient] [credits]",
+		Use:   "send [amount] [batch_denom] [recipient]",
+		Short: "Sends credits from the transaction author (--from) to the recipient",
+		Long: `Sends credits from the transaction author (--from) to the recipient.
+
+Parameters:
+  amount: amount to send
+  batch_denom: batch denomination
+  recipient: recipient address`,
+		Args: cobra.ExactArgs(3),
```

Comment:
we can populate the `Example` field to the `Command` struct here with something like `Example: "regen tx ecocredit send 20 regen18xvpj53vaupyfejpws5sktv5lnas5xj2phm3cf C01-001-20200403-20210405-001 --from myKey`